sys/include/net/gnrc/neterr.h

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (C) 2015 Martine Lenders <mlenders@inf.fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_gnrc_neterr Error reporting
+ * @ingroup     net
+ * @brief       Allows for asynchronous error reporting in the network stack.
+ * @{
+ *
+ * @file
+ * @brief   Error reporting definitions.
+ *
+ * @author  Martine Lenders <mlenders@inf.fu-berlin.de>
+ */
+#ifndef GNRC_NETERR_H_
+#define GNRC_NETERR_H_
+
+#include <errno.h>
+#include <stdint.h>
+
+#include "kernel_types.h"
+#include "msg.h"
+#include "net/gnrc/pkt.h"
+#include "thread.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   @ref core_msg type for reporting an error.
+ */
+#define GNRC_NETERR_MSG_TYPE        (0x0206)
+
+/**
+ * @brief   Error code for success.
```

Comment:
How? For me it makes perfect sense, honostly. It's a value in the class of error codes, that indicates success (no error).